fqtest.h

```diff
@@ -86,8 +86,29 @@ int retVal;
  return 1;
 
 }	
+
+
+
+
+void testpromotecoflows(struct Qdisc *sch, struct fq_sched_data *q )
+{
+
+
+    //struct fq_sched_data *p = qdisc_priv(sch);
+    struct fq_flow *f;
```

Comment:
These are just pointers, you may want to initialize `struct fq_flow f` and pass a pointer `&f`